e2e/src/usecases/WalletConnectV2.js

```diff
@@ -17,17 +19,15 @@ import { enterPinUiIfNecessary, navigateToSettings, sleep } from '../utils/utils
 import jestExpect from 'expect'
 
 const dappName = 'WalletConnectV2 E2E'
-
-const kitUrl = process.env.FORNO_URL || 'https://alfajores-forno.celo-testnet.org'
-const kit = newKit(kitUrl)
+const transportUrl = process.env.FORNO_URL || 'https://alfajores-forno.celo-testnet.org'
 
 const walletAddress = (
   process.env.E2E_WALLET_ADDRESS || '0x6131a6d616a4be3737b38988847270a64bc10caa'
 ).toLowerCase()
 
 async function formatTestTransaction(address) {
   try {
-    const response = await fetch(kitUrl, {
+    const response = await fetch(transportUrl, {
```

Comment:
i wonder if we can use [getBlockTransactionCount](https://viem.sh/docs/actions/public/getBlockTransactionCount#getblocktransactioncount) instead? it is less code.